docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+
+Questions from the CLI
+======================
+
+So far we've only learned how to say yes or no to the CLI's questions 
+when we migrate a schema, but quite a few other options are presented 
+when the CLI asks us a question:
+
+.. code-block::
+
+    did you create object type 'default::PlayerCharacter'? [y,n,l,c,b,s,q,?]
+    > y
+
+The choices ``y`` and ``n`` are obviously yes and no, and you can 
+probably guess that ``?`` will pull up a help menu, but the others 
+aren't so clear. Let's go over every option to make sure we understand 
+them.
+
+``y`` (or ``yes``)
+------------------
+
+This will accept the proposed change and move on to the next step. 
+If it's the last proposed change, the migration will now be complete.
+
+``n`` (or ``no``)
+-----------------
+
+This will reject the proposed change. At this point, the migration 
+tool will try to suggest a different change if it can, but it won't 
+always be able to do so.
+ 
+We can see this behavior with the same tiny schema change we made 
+above where we changed a property name from ``name`` to ``nam``. In 
+the output below, we see the following:
+
+- The CLI first asks us if we renamed the property, to which we say no.
+- It then tries to confirm that we have altered the ``User`` type. 
+  We say no again. 
+- The CLI then guesses that maybe we are dropping and creating the 
+  whole ``User`` type instead. This time, we say yes.
+- It then asks us to confirm that we are creating a ``User`` type, 
+  since we have decided to drop the existing one.
+
+But if we say no again to this question, the CLI will throw its hands 
+up and tell us that it doesn't know what we are trying to do because 
+there is no way left for it to migrate to the schema that we have 
+told it to move to.
+
+Here is the output:
+
+.. code-block::
+
+    did you rename property 'name' of object type 'default::User' 
+    to 'nam'?
+    [y,n,l,c,b,s,q,?]
+    > n
+    did you alter object type 'default::User'? [y,n,l,c,b,s,q,?]
+    > n
+    did you drop object type 'default::User'? [y,n,l,c,b,s,q,?]
+    > y
+    did you create object type 'default::User'? [y,n,l,c,b,s,q,?]
+    > n
+    Error executing command: EdgeDB could not resolve migration with 
+    the provided answers. Please retry with different answers.
+
+``l`` (or ``list``)
+-------------------
+
+This is used to see (list) the actual DDL statements that are being proposed.
+When asked the question ``did you alter object type 'default::User'?``
+in the example above, we might be wondering exactly what changes will 
+be made here. How exactly does the database intend to alter the ``User`` 
+type if we say yes? Simply clicking ``l`` will show it:
+
+.. code-block::
+
+    The following DDL statements will be applied:
+      ALTER TYPE default::User {
+          CREATE PROPERTY nam: std::str;
+          DROP PROPERTY name;
+      };
+
+This shows us clear as day that saying ``yes`` will result in creating 
```

Comment:
```suggestion
This shows us clear as day that saying "yes" will result in creating 
```

It looks like in most instances up to now, you're not styling "yes" as inline code, which I think is correct since the actual response is simply `y`. I've made this match those other instances but also surrounded it with quotes to reflect this is our reply like I have with the others.